css/gridfield-button-link.css

```diff
@@ -0,0 +1,7 @@
+.gridfield-button-link span.ui-icon {
+    display: inline-block;
+    vertical-align: bottom;
+    margin-right: 5px;
+    position: relative;
+    top: -1px;
+}
```

Comment:
Can you use the sitesummary.css file now?

Reply:
Actually, that bit of CSS is not needed after all, so I removed it.